api/projects/models.py

```diff
@@ -106,6 +106,7 @@ class Project(LifecycleModelMixin, SoftDeleteExportableModel):
         default=30,
         help_text="Number of days without modification in any environment before a flag is considered stale.",
     )
+    minimum_change_request_approvals = models.IntegerField(blank=True, null=True)
```

Comment:
This will also need adding to relevant serializer(s).

Reply:
Alright I've added it to `ProjectListSerializer` which looks like it will include the field to `ProjectRetrieveSerializer` and other serializers.